config/environments/production.rb

```diff
@@ -1,12 +1,8 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
-  if HostingEnvironment.review?
```

Comment:
not sure about this, will check tomorrow

Reply:
I don't think we create a read rep for review apps?